app/models/menu/menu.rb

```diff
@@ -285,10 +285,10 @@ def hmis_menu
     GrdaWarehouse::DataSource.hmis.distinct.map do |hmis_ds|
       Menu::Item.new(
         user: user,
-        visible: ->(_user) { HmisEnforcement.hmis_enabled? },
+        visible: ->(user) { user.any_hmis_access? },
```

Comment:
is this ok performance-wise @eanders?

I don't like advertising this link to warehouse-only users, since they technically can go log in but won't see anything.

Reply:
Should be fine, it will add a single exists query.  maybe we should memoize the result.  I'll note that in the method.